### ISSUE TITLE
Implementation Request: dropout between gates in LSTM

### ISSUE THREAD

**Issue by supakjk** [Element-Research/rnn#335]
http://arxiv.org/abs/1512.05287 shows the usage of dropout between gates in LSTM. It would be great if we can also easily use such "inside" dropouts in rnn modules like FastLSTM.
Keras already supports it https://keras.io/layers/recurrent/

**Comment by jnhwkim** [comment]
@supakjk bayesian dropout on GRU is implemented but yet to LSTM. However, I've already implement it recently for my experiment. When the test is done, I'll send a PR.

**Comment by supakjk** [comment]
It would also be great if we can easily set the initial forget gate bias.

**Comment by jnhwkim** [comment]
Can you check [the implementation](https://github.com/jnhwkim/rnn/commit/5c49ab69f1b69b52fa41e24fb534d4ad1fab15a2)? You can use nn.LSTM or nn .FastLSTM . 

You can access submodule via `self.recurrentModule`.

**Comment by jnhwkim** [comment]
Refer to #382.
